app/controllers/account/home_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
       @loans = current_member.loans.checked_out
       @ready_holds_count = current_member.holds.active.started.count
       @waiting_holds_count = current_member.holds.active.waiting.count
+      @recent_approvals = current_member.borrow_policy_approvals.approved.where(updated_at: 2.weeks.ago..)
```

Comment:
Since we have plenty of room on this page, I say we just leave the approval in place all the time rather than scoping to only recent approvals